src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1146,10 +1147,12 @@ protected override void GetItem(string path)
                         }
                     }
                 }
+#endif
 
                 FileSystemInfo result = GetFileSystemItem(path, ref isContainer, false);
                 if (result != null)
                 {
+#if !UNIX
                     // If we want to retrieve the file streams, retrieve them.
                     if (retrieveStreams)
```

Comment:
And when running `Get-Content -Path file::stream` on Linux, it probably should throw an error message about "alternate stream is not supported" instead of working unexpectedly or fail with other errors.